interfaces/session_cost_calculator.yaml

```diff
@@ -0,0 +1,11 @@
+description: >-
+  This interface publishes the running or finished session costs. This interface
+  provides cost of one session. If we have more than one outlet, we need to 
+  instantiate this interface for each outlet.
+vars:
+  session_cost:
+    description: >-
+      Session cost object containing the total cost of the session and a list
+      of chunks
+    type: object
+    $ref: session_cost#/SessionCost
```

Comment:
The file is called session_price - and here you refer to session_cost. I think it should be session_cost as we typically call the file name after the "main" message

Reply:
Fixed